### PR TITLE
Polish translation of Tour of Scala: Tuples 

### PR DESCRIPTION
**Pull request by kamilduda**
Add translation based on English version

### CODE REVIEW

File: _pl/tour/tuples.md [hunk 1]
```diff
@@ -1,17 +1,93 @@
...
 previous-page: traits
 
 ---
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
+W Scali, krotka (ang. tuple) to klasa, która przechowuje elementy różnych typów.
+Krotki są niezmienne.
+
+Krotki przydają się, kiedy, aby funkcja zwróciła jednocześnie wiele wartości.
```
Comment (jstefanska) [inline _pl/tour/tuples.md]:
```suggestion
Krotki przydają się, gdy chcemy, żeby funkcja zwróciła jednocześnie wiele wartości.
```

File: _pl/tour/tuples.md [hunk 2]
```diff
@@ -1,17 +1,93 @@
...
+Powyższy kod tworzy krotkę zawierającą element typu String oraz typu Int.
+
+W Scali krotki reprezentowane są przez szereg klas: Tuple2, Tuple3 itd. aż do Tuple22.
+Za każdym razem, kiedy tworzymy krotkę zawierającą _n_ elementów (_n_ musi zawierać się w przedziale od 2 do 22), Scala tworzy instancję jednej z odpowiadających klas z grupy TupleN i parametryzuje ją odpowiednimi wartościami.
+W ww. przykładzie jest to `Tuple2[String, Int]`.
+
+## Dostęp do elementów
+
+Krotka zapewnia dostęp do swoich elementów z użyciem składni podkreślnika (underscore).
+`tuple._n` odwołuje się do n-tego elementu w kolejności (pod warunkiem że dana krotka zawiera tyle elementów).
```
Comment (jstefanska) [inline _pl/tour/tuples.md]:
```suggestion
`tuple._n` odwołuje się do n-tego elementu w kolejności (pod warunkiem, że dana krotka zawiera tyle elementów).
```

File: _pl/tour/tuples.md [hunk 3]
```diff
@@ -1,17 +1,93 @@
...
+  println(a * b)
+}
+```
+{% endscalafiddle %}
+
+Wartość `()` typu `Unit` jest koncepcyjnie taka sama jak wartość `()` typu `Tuple0`.
+Może być tylko jedna wartość tego typu, ponieważ nie zawiera żadnych elementów.
+
+Użytkownicy mogą czasami mieć trudności z wyborem pomiędzy krotkami (tuple) i klasami przypadków (case class).
+Z reguły, klasy przypadków są preferowane w przypadkach, kiedy elementy niosą ze sobą jakieś większe znaczenie.
```
Comment (jstefanska) [inline _pl/tour/tuples.md]:
```suggestion
Zazwyczaj klasy przypadków są preferowane wtedy, kiedy elementy niosą ze sobą jakieś większe znaczenie.
``

**Comment by kamilduda** [comment]
@jstefanska fixed

**Comment by SethTisue** [comment]
awesome, thanks!

**Pull request merged**